src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -130,7 +130,8 @@ internal static void SetElementTransform(IntPtr htmlId, Matrix3x2 matrix)
 				SetStyles(
 					htmlId,
 					new[] { ("transform", native.ToStringInvariant()) },
-					true
+					setAsArranged: true,
+					clipToBounds: requiresClipping
```

Comment:
I don't understand why we have to provide this value each time we apply a style. We cannot replicate it in the DOM?

Reply:
@carldebilly

Reply:
This does not change much with what is done already to fix the issue, but I agree, it could probably be synchronized in the DOM instead.

Reply:
Yeah I agree that the flag was not added by this PR, it was more to understand if we had a good reason to this like that or if was just a quick fix that was forgotten before merge and which drives to this kind of error :)

Reply:
Yeah there's no need to set the clipping mode every time `SetStyles()` is called, I can refactor this.

Reply:
Fixed